### PR TITLE
provider/openstack: changes to packer to pass in image id instead of image name

### PR DESCRIPTION
**Pull request by drmaas**

### CODE REVIEW

**Comment by drmaas** [comment]
@spinnaker/reviewers

File: rosco-web/config/rosco.yml [hunk 1]
```diff
@@ -255,11 +255,11 @@ openstack:
       virtualizationSettings:
       - region: r1
         instanceType: smem-2vcpu
-        sourceImageName: ubuntu-latest
+        sourceImageId: b12f2467-cfca-4a65-b29b-d11b8c46818d
```
Comment (duftler) [inline rosco-web/config/rosco.yml]: Would it make sense to drop a link here so a reader knows where this value is retrieved from? And does this have to be kept in sync somehow?

**Comment by duftler** [comment]
LGTM after addressing comment.
Will need a rebase.

**Comment by drmaas** [comment]
Link, as in comment?

Not sure what you mean about keeping it in sync - name and id are 2 of the attributes available for an image, and everything we're doing in spinnaker is operating on uuids.

**Comment by duftler** [comment]
Yup, I meant as a comment.
W.r.t. keeping the id's in sync, I had something like this in mind: https://github.com/spinnaker/rosco/blob/master/rosco-web/config/rosco.yml#L52-L53
Are these id's placeholders, durable/will never change, or something else?

**Comment by drmaas** [comment]
Added comment.

**Comment by duftler** [comment]
Thanks.

**Pull request merged**